src/qminer/qminer_core.h

```diff
@@ -3219,46 +3219,45 @@ namespace TStreamAggrOut {
 
 	class IInt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual int GetInt() const = 0;
 	};
 
 	class IFlt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual double GetFlt() const = 0;
 	};
 
+	class ISparseVec {
```

Comment:
There is a problem later on in the chain, however - collision of methods in `TNodeJsFuncStreamAggr` - method `GetVal` comes from `IFltTm` and `ISparseVecTm` and they have different signatures...